src/test/tools/gran_detecto/gran_detecto.c

```diff
@@ -328,7 +328,7 @@ main(int argc, char *argv[])
 		}
 	}
 #endif
-	struct tool_ctx ctx = {0};
+	struct tool_ctx ctx;
```

Comment:
what was the problem?

Reply:
11:02:25,462 INFO  - gran_detecto.c:331:26: error: missing field 'probe_file_path' initializer [-Werror,-Wmissing-field-initializers]
11:02:25,462 INFO  - struct tool_ctx ctx = {0};
11:02:25,462 INFO  - ^
11:02:25,462 INFO  - 1 error generated.